### ISSUE TITLE
[Discord] Support for ephemeral commands

### ISSUE THREAD

**Issue by rexor12** [rexor12/holobot#122]
In some cases I may want a command's response(s) to be ephemeral (that is, only visible to the user who invoked the command).

For example, as an administrator, while setting up the bot for the server I may want the related commands to be visible to me only.

Introduce support for this so that
- use case 1: _the developer_ can specify if a command is ephemeral
- use case 2: _the developer_ can introduce a command parameter using which _the user_ can specify if they want the command to be ephemeral

May be more easily feasible after #118.

**Comment by rexor12** [comment]
Ephemeral commands have been introduced by #118 and it supports use case 1. The problem with use case 2 is that the initial response (for deferred responses) is sent before the interactable is processed, therefore the generic interactable processor would need to figure out if the initial response should be ephemeral in case there is one. Since the workflow and interactable are already known at this point, it may be possble to introduce an `is_ephemeral(self, **arguments)` or similar method to either the workflow or the interactable that interprets the arguments and determines if the response is going to be ephemeral.

**Comment by rexor12** [comment]
#350 (#351) has covered this issue as well. Interaction replies can now specify whether they prefer to be ephemeral. If the response is deferred, the initial response's ephemeral option determines the final response's ephemeral option.
